cmake/kconfig.cmake

```diff
@@ -148,12 +138,8 @@ foreach(merge_config_input ${merge_config_files} ${DOTCONFIG})
   set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${merge_config_input})
 endforeach()
 
-message(STATUS "Generating zephyr/include/generated/autoconf.h")
 execute_process(
-	COMMAND ${KCONFIG_CONF}
-    --silentoldconfig
-    ${KCONFIG_ROOT}
-  WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/kconfig
+COMMAND ${CMAKE_COMMAND} -E remove_directory ${PROJECT_BINARY_DIR}/kconfig
```

Comment:
This is a leftover from debugging, to check that the `kconfig/` folder, which is no longer being generated, is not required. Can be removed.

Reply:
+1 to removing it.